File: Array/SquaresOfSortedArray/Solution.1.js
const sorted = function (arr) {
    for (let i = 0; i < arr.length; i++) {
        arr[i] = arr[i] ** 2
    }

    arr.sort((x, y) => x - y)
    return arr
}

let sortArray = [-5, -4, -3, -2, 3, 4, 5, 6]

sorted(sortArray)
console.log(sortArray);

File: DSA/DataStructures/Array/9. MinCost/Solution1.java
import java.util.Arrays;
public class Solution1 {

    public static void minCost(int[] arr) {

        int totalSum=0;

        for(int a : arr) 
            totalSum+=a;

        Arrays.sort(arr);

        int minCost = 0;
        int n = arr.length;

        for(int i=n-1; i>-1; i--) {
            minCost+=totalSum;
            totalSum -= arr[i];            
        }

        System.out.println("Min Cost is : " + minCost);
    }

    public static void main(String args[]) {
        int[] nums = {3,2,1};

        minCost(nums);
    }
}